.github/workflows/tests.yml

```diff
@@ -192,16 +192,17 @@ jobs:
             exit "0"
           fi
 
-  check-snapshots:
-    name: "🔍 Check for valid snapshot urls"
-    runs-on: ubuntu-20.04
-    steps:
-      - name: Check out code into the Go module directory
-        uses: actions/checkout@v4
-        with:
-          ref: ${{ github.event.pull_request.head.sha }}
-      - name: Check for valid snapshot urls
-        run: ./tools/check-snapshots --errors-only .
+  # TODO: enable again once enumerating rpmrepo does not fail anymore
```

Comment:
What do you mean by "fail anymore" ? 

I've seen this timing out on occasion but restarting the job usually works fine for me. Is there a more serious problem than that?

Reply:
Also FTR TODO comments tend to get forgotten if we don't have tickets for them.

Reply:
We probably need to clean up some old snapshots, because the timeout limits are at the maximum.

Reply:
And this is now failing consistently also in the osbuild repo... restarting won't help.

Reply:
I think it's failing more often than it's passing now.  It's getting increasingly unreliable and I've had occasions where I had to retry it 4-5 times to get it to pass.

Reply:
I created https://issues.redhat.com/browse/COMPOSER-2263

Reply:
@atodorov are you OK with keeping it as is with the ticket?